panda/src/gobj/texturePool.cxx

```diff
@@ -66,13 +68,87 @@ register_texture_type(MakeTextureFunc *func, const string &extensions) {
  * Records a TexturePoolFilter object that may operate on texture images as
  * they are loaded from disk.
  */
+bool TexturePool::
+ns_register_filter(TexturePoolFilter *tex_filter) {
+  MutexHolder holder(_filter_lock);
+
+  // Make sure we haven't already registered this filter.
+  if (find(_filter_registry.begin(), _filter_registry.end(), tex_filter) != _filter_registry.end()) {
```

Comment:
I'm sure there's a `using std::find;` somewhere that makes this work, I just can't find it… is a thirdparty library pulling this in?  Maybe use `std::find` to be safe.